src/applications/bmqtool/m_bmqtool_inpututil.cpp

```diff
@@ -321,6 +321,40 @@ bool InputUtil::populateSubscriptions(bmqt::QueueOptions*              out,
     return !failed;
 }
 
+bool InputUtil::populateSubscriptions(bmqt::QueueOptions* out,
+                                      int                 autoPubSubModulo,
+                                      const char*       autoPubSubPropertyName,
+                                      bslma::Allocator* allocator)
+{
+    BSLS_ASSERT_SAFE(out);
+
+    bool failed = false;
+    for (int i = 0; i < autoPubSubModulo; ++i) {
```

Comment:
Do you think it's worth adding a reasonable upper limit for `autoPubSubModulo`, so we don't use all RAM? 1kk?